packages/story-editor/src/components/library/panes/pageTemplates/karma/defaultPageTemplates.karma.js

```diff
@@ -215,6 +213,10 @@ describe('CUJ: Page Templates: Creator can Apply a Default Page Template', () =>
       await fixture.events.keyboard.press('right');
       await fixture.events.keyboard.press('right');
       await fixture.events.keyboard.press('Enter');
+
+      // allow filtered templates to be removed from DOM
+      await fixture.events.sleep(500);
```

Comment:
To be honest I couldn't get this to fail regularly. When it did fail, it was because all the templates were still present when the below expect would run. Adding the sleep to give it some time for moving on seemed to help as I couldn't get it to fail once I did so.

Reply:
I wonder if just increasing the `timeout` option param on the `waitFor` below would have a similar effect on making this robust without always making this test take an extra 500ms. Also tho I feel like `waitFor` doesn't always behave as expected so understand if this needs to be a `sleep` call

Reply:
Totally agree. The `sleep` feels cheap. I'll try a few more test rounds with the `waitFor` cause I do like that approach better.